extensions/rich_text_components/protractor.js

```diff
@@ -22,6 +22,7 @@ var RICH_TEXT_COMPONENTS = {
   Image: require('./Image/protractor.js'),
   Link: require('./Link/protractor.js'),
   Math: require('./Math/protractor.js'),
+  Skillreview: require('./Skillreview/protractor.js'),
```

Comment:
Could you please add a backend test that iterates through all our RTEs and checks that a line similar to this exists in protractor.js? This helps ensure that, if someone adds a component in the future, they remember to add the e2e test as a matter of course.

Thanks!

Reply:
Done, added a test to components_test that verifies this. Though, it takes the raw text inside this and matches the constants, is that fine?

Reply:
Yep, regex approach is fine (but see my other comments). Thanks!